NewForce-Cohort-4/tabloid-full-stack-team-rose
# Af postdelete

Description
Added the ability for a logged in user to delete that user's posts.

Fixes # (issue)

Type of change
 New feature (non-breaking change which adds functionality)

Testing Instructions
pull down af-postdelete. Start the API. npm start that react app. login with an actual user. go to all posts. click on post details of a post your user didn't create. you should see a not authorized button. go back to all posts. find a post created by your user. click post details. click delete button. click cancel and be redirected to all posts. find a post created by your user. click post details. click delete. click confirm delete. the post is deleted and you are redirected to all posts. 

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
